accepted/0000-subscription-matching-in-public-clouds.md

```diff
@@ -0,0 +1,194 @@
+- Feature Name: Subscription matching in public clouds
+- Start Date: 2019-08-14
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC set the guides for supporting "Subscription Matching" on virtual instances created on Public Clouds like AWS, Azure or Google Compute Engine. Exploring and gatehering virtual instances from those Public Clouds enables "Subscription Matching" to properly match subscriptions for "1-2 Virtual Machines".
```

Comment:
What about unlimited VMs subscriptions? If they are disallowed by T&C, let's state that clearly

Reply:
Currently 1-2 Subscriptions and per instance Subscriptions are the only which can be used in Public Cloud.
Unlimited Virtualization cannot be used. We can add this to the RFC.